Classes/BITHockeyHelper.m

```diff
@@ -174,10 +174,10 @@ void bit_fixBackupAttributeForURL(NSURL *directoryURL) {
       if ([directoryURL getResourceValue:&appSupportDirExcludedValue forKey:NSURLIsExcludedFromBackupKey error:&getResourceError] && appSupportDirExcludedValue) {
         NSError *setResourceError = nil;
         if(![directoryURL setResourceValue:@NO forKey:NSURLIsExcludedFromBackupKey error:&setResourceError]) {
-          BITHockeyLogError(@"ERROR: Error while setting resource value: %@", setResourceError.localizedDescription);
+          BITHockeyLogDebug(@"ERROR: Could not set resource value: %@", setResourceError.localizedDescription);
```

Comment:
Shouldn't be "DEBUG" in the message too instead of ERROR?
Also could you figure out why we enter this second if since the get succeeded?